js/packages/core/package.json

```diff
@@ -53,6 +53,8 @@
     "tsup": "^6.0.1"
   },
   "dependencies": {
+    "@types/events": "^3.0.0",
```

Comment:
Do we use types/events here? It seems like we're manually declaring our type layer over the events package

Reply:
Good call. Tested this out and we don't need it. 👍